frontend/packages/kubevirt-plugin/src/components/vms/vm-details.tsx

```diff
@@ -78,11 +81,24 @@ export const VMDetails: React.FC<VMDetailsProps> = (props) => {
   const vmiLike = kindObj === VirtualMachineModel ? vm : vmi;
   const vmServicesData = getServicesForVmi(getLoadedData(props.services, []), vmi);
   const canUpdate = useAccessReview(asAccessReview(kindObj, vmiLike || {}, 'patch')) && !!vmiLike;
+  const [guestAgentInfo] = useGuestAgentInfo({ vmi });
+
+  const OSMismatch =
```

Comment:
nitpick: Can we call this `OSMismatchExists`?

Reply:
+1 fixed